src/jvmTest/kotlin/fr/acinq/lightning/Node.kt

```diff
@@ -231,10 +232,7 @@ object Node {
                     }
                 }
                 install(ContentNegotiation) {
-                    json(Json {
-                        serializersModule = Serialization.lightningSerializersModule
-                        allowStructuredMapKeys = true
-                    })
+                    json(JsonSerializers.json)
```

Comment:
It seems that now API calls to `/channels` work but not calls to `/channels/{channelId}`:

```
Serializer for class 'Normal' is not found.
Mark the class as @Serializable or provide the serializer explicitly.
```

Reply:
There are no tests on this thing, I should manually test all endpoints I guess.

Reply:
Fine with me, it was clumsy to maintain (copying parameters back from phoenix,...) and I was the only one to use it. It means we can also remove `ktor` from our JVM test dependencies.

Reply:
I cleaned up dependencies in [09e4957](https://github.com/ACINQ/lightning-kmp/pull/391/commits/09e4957da11e7f087e2f13c303f46e51d8a285b0).